src/Features/Core/Portable/EditAndContinue/AbstractEditAndContinueAnalyzer.cs

```diff
@@ -6779,19 +6772,23 @@ private static bool HasGetHashCodeSignature(IMethodSymbol method)
     internal TestAccessor GetTestAccessor()
         => new(this);
 
-    internal readonly struct TestAccessor(AbstractEditAndContinueAnalyzer abstractEditAndContinueAnalyzer)
+    internal readonly struct TestAccessor(AbstractEditAndContinueAnalyzer analyzer)
     {
-        private readonly AbstractEditAndContinueAnalyzer _abstractEditAndContinueAnalyzer = abstractEditAndContinueAnalyzer;
+        internal Action<SyntaxNode>? FaultInjector
+        {
+            get => analyzer._testFaultInjector = FaultInjector;
```

Comment:
Does `FaultInjector` here bind in some way I don't see, or is this an infinite loop?